ui/src/components/RoleSelectScreen.tsx

```diff
@@ -63,7 +58,7 @@ const RoleSelectScreen: React.FC<Props> = ({ onLogout }) => {
 
         setLoading(false);
         setRole(undefined);
-        history.push(`/role/${role.toLowerCase()}`);
+        history.push(`/role/${role.replace('Role', '').toLowerCase()}`);
```

Comment:
I believe it is better if you trim the suffix `Role` than just replace the word `Role` with the empty string

Reply:
Changed to use `slice`.